test/AutoRest.TestServer.Tests/Mgmt/TestProjects/MgmtListMethods.cs

```diff
@@ -19,11 +20,34 @@ public class MgmtListMethods : TestProjectTests
         [TestCase("TheExtensionFakeContainer", "ListFeatures", false)]
         [TestCase("TheExtensionFakeOperations", "ListUsages", true)]
         [TestCase("TheExtensionFakeOperations", "ListFeatures", true)]
-        public void ValidateListOfNonResourceMethod(string operationOrContainerName, string methodName, bool exist)
+        [TestCase("SubscriptionExtensions", "ListFakes", true)]
+        [TestCase("FakeContainer", "ListFakes", false)]
+        [TestCase("FakeOperations", "ListFakes", false)]
+        [TestCase("SubscriptionExtensions", "ListByLocations", true)]
+        [TestCase("FakeContainer", "ListByLocations", false)]
```

Comment:
shouldn't the name of the method still be List with just an overload?

Reply:
Renamed it to List<ResourceName>sByLocation, since the parameter count and types are same for ListAll and ListByLocation.